pages/tutorials/3.0/system/time.md

```diff
@@ -47,11 +60,14 @@ bool b1 = (t1 == t2);
 bool b2 = (t3 > t4);
 ```
 
+`sf::Time` is fully `constexpr`-enabled so all of these operations can be performed in a compiletime context.
+
 ## Measuring time
 
 Now that we've seen how to manipulate time values with SFML, let's see how to do something that almost every program needs: measuring the time elapsed.
 
-SFML has a very simple class for measuring time: [`sf::Clock`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Clock.php "sf::Clock documentation"). It only has two functions: `getElapsedTime`, to retrieve the time elapsed since the clock started, and `restart`, to restart the clock.
+SFML has a very simple class for measuring time: [`sf::Clock`](https://www.sfml-dev.org/documentation/3.0.0/classsf_1_1Clock.php "sf::Clock documentation").
+It only has two functions: `getElapsedTime`, to retrieve the time elapsed since the clock started, and `restart`, to restart the clock.
```

Comment:
This sentence is out of date. I need to add a sentence about the new `sf::Clock` functions.

This brings up a larger point though, do we want these docs to highlight features new in v3? For example we could have some colored text saying "_New in version 3!_". Doing so may be useful but will make it more effortful to update all of these docs.